packages/client/modules/userDashboard/components/OrgBilling/BillingForm.tsx

```diff
@@ -63,33 +85,54 @@ const BillingForm = (props: Props) => {
   const [isLoading, setIsLoading] = useState(false)
   const [isPaymentSuccessful, setIsPaymentSuccessful] = useState(false)
   const atmosphere = useAtmosphere()
-  const {onError, onCompleted} = useMutationProps()
+  const {onError} = useMutationProps()
   const [errorMsg, setErrorMsg] = useState<null | string>()
 
   const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault()
     if (!stripe || !elements) return
     setIsLoading(true)
     if (errorMsg) setErrorMsg(null)
-    const {setupIntent, error} = await stripe.confirmSetup({
-      elements,
-      redirect: 'if_required'
+    const cardElement = elements.getElement(CardElement)
+    if (!cardElement) return
+    const {paymentMethod, error} = await stripe.createPaymentMethod({
+      type: 'card',
+      card: cardElement
     })
-    setIsLoading(false)
+
     if (error) {
       setErrorMsg(error.message)
       return
     }
-    const {payment_method: paymentMethodId, status} = setupIntent
-    if (status === 'succeeded' && typeof paymentMethodId === 'string') {
+
+    const handleCompleted = async (res: UpgradeToTeamTierMutation$data) => {
+      const {upgradeToTeamTier} = res
+      const stripeSubscriptionClientSecret =
+        upgradeToTeamTier?.organization?.stripeSubscriptionClientSecret
+      if (!stripeSubscriptionClientSecret) {
+        setIsLoading(false)
+        return
+      }
+      const {error} = await stripe.confirmCardPayment(stripeSubscriptionClientSecret)
+      if (error) {
+        setErrorMsg(error.message)
+        setIsLoading(false)
+        return
+      }
+      setIsLoading(false)
       setIsPaymentSuccessful(true)
-      UpgradeToTeamTierMutation(atmosphere, {orgId, paymentMethodId}, {onError, onCompleted})
     }
+
+    UpgradeToTeamTierMutation(
+      atmosphere,
+      {orgId, paymentMethodId: paymentMethod.id},
+      {onError, onCompleted: handleCompleted}
+    )
   }
 
   return (
     <StyledForm id='payment-form' onSubmit={handleSubmit}>
-      <PaymentElement id='payment-element' options={{layout: 'tabs'}} />
```

Comment:
I'm surprised that there is no 3d secure support. Not sure I understand why.
I'm [reading](https://stripe.com/docs/payments/payment-card-element-comparison) about PaymentElement, stripe says it is recommended solution, and shows 3d secure support

<img width="1150" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/412226d0-eb92-4d4f-b7a9-478e14ac461e">

Reply:
You can use `PaymentElement` with 3D Secure cards, but it looks like you can't with the `createPaymentMethod` method: https://stripe.com/docs/js/payment_methods/create_payment_method#stripe_create_payment_method-paymentMethodData-card

We need to use `createPaymentMethod` to create the payment method on the client using the card details the user inputs, then create the subscription on the server using those card details, and then confirm the card payment on the client once the 3D Secure card has been authorised by the user and we have the subscription client secret.